bindings/wasm/native/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "test-iota-client-wasm",
```

Comment:
And this would be called @iota/cllient-wasm then :D (see my comment on "native/Cargo.toml"

Reply:
Yeah `@iota/client-wasm` sounds good I think for the release, but need to check with the others